Connot find module 'sass'

**Describe the bug**
I cannot compile the code 
**To Reproduce**
Steps to reproduce the behavior:
- I build the project scaffolding with vue-ui
![image](https://user-images.githubusercontent.com/5508735/100878440-7b246700-34aa-11eb-9533-48ef8a967ad7.png)
- compilation fails
![image](https://user-images.githubusercontent.com/5508735/100878488-8d9ea080-34aa-11eb-86bb-cccdcdbb5db1.png)

**Expected behavior**
Looks to be missing a module but it's there ? 
**Screenshots**
See above 
 - Browser Chrome on MacBook
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

Failed to compile.

./node_modules/@grapoza/vue-tree/components/TreeViewNode.vue?vue&type=script&lang=js& (./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./node_modules/@grapoza/vue-tree/components/TreeViewNode.vue?vue&type=script&lang=js&)
Module not found: Error: Can't resolve '../objectMethods' in '/Users/mg61dd/workspace/P00002-lieven-playground/src/main/resources/www/browse/node_modules/@grapoza/vue-tree/components'

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -19,7 +19,7 @@ build_script:
 
 after_build:
   # Copy anything that goes into the final package into '/package'
-  - sh: mkdir -p package && cp dist/*.js dist/*.css index.js package.json LICENSE README.md package
+  - sh: mkdir -p package && cp dist/*.js dist/*.css index.js package.json LICENSE README.md src/objectMethods.js package
   - sh: mkdir -p package/components && cp src/components/*.vue package/components
   - sh: mkdir -p package/mixins && cp src/mixins/*.js package/mixins
   - sh: mkdir -p package/enums && cp src/enums/*.js package/enums

